org.eclipse.winery.topologymodeler.ui/src/app/palette/palette.component.ts

```diff
@@ -107,7 +107,13 @@ export class PaletteComponent implements OnInit, OnDestroy {
      * Angular lifecycle event.
      */
     ngOnInit() {
-        // console.log(this.entityTypes);
+    }
+
+    /**
+     * Angular lifecycle event.
+     */
+    ngAfterViewInit() {
+        console.log(this.entityTypes);
```

Comment:
Please remove this method, if it isn't essential.

Reply:
done